src/components/sidebar/labeled-messages/Row.js

```diff
@@ -124,7 +128,7 @@ export default class Row extends PureComponent {
           key={`${message.id}-message`}
           intl={intl}
           className={classes.message}
-          onSelect={onSelect}
+          onSelect={this.onSelectMessage}
```

Comment:
isn't onSelect already passing message object? I think you don't need additional wrapper with onSelectMessage